run-automation-test.md

```diff
@@ -0,0 +1,128 @@
++ [2. Configure automation test script](#2-config-automation-test-script)
++ [3. Run automation test on Kobiton devices](#3-run-automation-test-on-kobiton-devices)
++ [4. Fetch test session data through Kobiton REST API](#4-fetch-test-session-data-through-kobiton-rest-api)
++ [5. Feedback](#5-feedback)
+## Prerequisites 
+- Java(JRE). Supported are:
+  + Oracle java 8 or higher
+  + OpenJDK 8
+- Kobiton account
+    >Please visit https://portal.kobiton.com/register to create new account.
+## 1. Config TeamCity and integrate with GitHub
+Firstly, let's assume you already have an empty GitHub repository for running automation test.
+
+This part will guide you to how to configure TeamCity to integrate with your automation test GitHub repository. Skip this step if you have already had TeamCity setup.
+ + For instruction on how to install and configure TeamCity server, follow [this guide](https://confluence.jetbrains.com/display/TCD18/Installing+and+Configuring+the+TeamCity+Server).
+
+Set up your project and link your app repository that we will test with Kobiton later.
++ Instruction on how to integrate GitHub with TeamCity can be found [here](https://confluence.jetbrains.com/display/TCD10/Integrating+TeamCity+with+VCS+Hosting+Services).
+ 
+>For more information about TeamCity can be found in [this tutorial](https://confluence.jetbrains.com/display/TCD10/TeamCity+Documentation).
+
+## 2. Configure automation test script
+This documentation uses NodeJS as the default language to demonstrate the guide. 
+### 2.1 Get Username and API key
+Go to https://portal.kobiton.com and login to your Kobiton account.
+- Username
+  + Click *user icon* -> **"Profile"** (you might find it in the top right corner)
+
+  ![get username](./assets/username.png )
+
+- API key 
+  + Click *user icon* -> **"Settings"**
+
+  ![get api key](./assets/api_key.png )
+
+### 2.2 Get desired capabilities
+The desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device.
+  + Click **Devices** in the top navigation bar.
+ 
+    ![click devices](./assets/devices.png )
+ 
+  + Hover over any device you want to test with and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+    ![click auto](./assets/click_auto.png )
+ 
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script, therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section.
+
+    ![click auto](./assets/automation.png ) 
+### 2.3 Configure automation test script
+Kobiton has already provided sample scripts for automation testing, visit [here](https://github.com/kobiton/samples) for reference. 
+
+In this example, we will be running Android App Testing, therefore, we will use `android-app-test` script.
```

Comment:
Move this line to step 3 of session 2.4.
As we are guiding the users to configure to run their own automation script, our sample is only be used **for reference**. Therefore, forcing them use `android-app-test` is kinda odd.